lcls_tools/superconducting/scLinac.py

```diff
@@ -1544,16 +1544,17 @@ def characterize(self):
         sleep(2)
 
         while self.characterization_running:
+            self.check_abort()
             print(
                 f"waiting for {self} characterization" f" to stop running",
                 datetime.now(),
             )
             sleep(1)
 
         if self.characterization_status == utils.CALIBRATION_COMPLETE_VALUE:
-            if (datetime.now() - self.characterization_timestamp).total_seconds() > 60:
+            if (datetime.now() - self.characterization_timestamp).total_seconds() > 300:
                 raise utils.CavityQLoadedCalibrationError(
-                    f"{self} characterization did not start"
+                    f"{self} characterization took longer than 5 minutes to run"
```

Comment:
waiting for 5min without any feedback on progress makes me guess that people will start saying 'the GUI is frozen'. Maybe 2 or 3minutes? Or can we think of any way to convey that things are actually happening?

Reply:
Well the thing is that right now it's just waiting indefinitely (not even just 5 minutes but however long it takes for characterization to run) in the `while self.characterization_running` section. This chunk is just to check that we for sure had a successful characterization within the last 5 minutes, in case it actually didn't start

Reply:
Oh right. Maybe the better wording then would be 'no valid characterization within the last 5 min'?